okhttp-graal/src/main/kotlin/okhttp3/graal/svm.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.graal
+
+import com.oracle.svm.core.annotate.Delete
+import com.oracle.svm.core.annotate.Substitute
+import com.oracle.svm.core.annotate.TargetClass
+import okhttp3.internal.platform.Android10Platform
+import okhttp3.internal.platform.AndroidPlatform
+import okhttp3.internal.platform.BouncyCastlePlatform
+import okhttp3.internal.platform.ConscryptPlatform
+import okhttp3.internal.platform.Jdk8WithJettyBootPlatform
+import okhttp3.internal.platform.Jdk9Platform
+import okhttp3.internal.platform.OpenJSSEPlatform
+import okhttp3.internal.platform.Platform
+
+@TargetClass(AndroidPlatform::class)
+@Delete
+class TargetAndroidPlatform {
```

Comment:
The goal of these are to get to the point that "allow-incomplete-classpath" isn't required.  But I didn't set that as complete goal yet.

Reply:
Currently would fail with 

```
com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: android.os.Build$VERSION. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
```